### PR TITLE
Fix vulnerabilities.sh when zero vulns

### PR DESCRIPTION
**Pull request by karlkfi**

### CODE REVIEW

**Review by nan-yu — APPROVED** [review]
/lgtm

File: scripts/vulnerabilities.sh [hunk 1]
```diff
@@ -99,7 +99,8 @@ for image in "${images[@]}"; do
   fixable=$(echo "${vulnerabilities}" |
     jq -r 'with_entries(select(.key == "CRITICAL" or .key == "HIGH" or .key == "MEDIUM")) | select(.vulnerability != {}) | map(map(select(.vulnerability.packageIssue[].fixAvailable == true)) | length) + [0] | add')
   vuln_map[${image}]=${fixable}
-  ((fixable_total += fixable))
+  # add vuln count to total, but don't fail if zero
+  ((fixable_total += fixable)) || true
```
[the remark this answers is not in the document]
Comment (karlkfi) [reply scripts/vulnerabilities.sh:103]: nvm. you are correct.

[the remark this answers is not in the document]
Comment (karlkfi) [reply scripts/vulnerabilities.sh]: Fixed & pushed

[the remark this answers is not in the document]
Comment (karlkfi) [reply scripts/vulnerabilities.sh]: I remembered incorrectly. The linter is ok with this. It was failing when I used `let` and told me to use (()) instead.

**Pull request merged**